doc/source/ray-observability/user-guides/configure-logging.md

```diff
@@ -138,11 +138,35 @@ The output is as follows:
 
 This feature is especially useful when importing libraries such as `tensorflow` or `numpy`, which may emit many verbose warning messages when imported. Configure this feature as follows:
 
-1. Set ``RAY_DEDUP_LOGS=0`` to disable this feature entirely.
-2. Set ``RAY_DEDUP_LOGS_AGG_WINDOW_S=<int>`` to change the agggregation window.
-3. Set ``RAY_DEDUP_LOGS_ALLOW_REGEX=<string>`` to specify log messages to never deduplicate.
-4. Set ``RAY_DEDUP_LOGS_SKIP_REGEX=<string>`` to specify log messages to skip printing.
-
+* Set ``RAY_DEDUP_LOGS=0`` to turn off this feature entirely.
+* Set ``RAY_DEDUP_LOGS_AGG_WINDOW_S=<int>`` to change the aggregation window.
+* Set ``RAY_DEDUP_LOGS_ALLOW_REGEX=<string>`` to specify log messages to never deduplicate.
+* Set ``RAY_DEDUP_LOGS_SKIP_REGEX=<string>`` to specify log messages to skip printing.
+    * Configure `RAY_DEDUP_LOGS_SKIP_REGEX` on the driver process before importing Ray.
```

Comment:
I'm wondering if the statement is true for all the variables above and not just `RAY_DEDUP_LOGS_SKIP_REGEX`. If that's the case, wondering if we should make the description more general? 

For reference, it seems that people are also having troubles with `RAY_DEDUP_LOGS` by looking at the thread here: https://discuss.ray.io/t/how-to-set-ray-dedup-logs-0/10465

Reply:
good point!